src/tests/global_variables/env_variables_defaults.py

```diff
@@ -70,6 +71,11 @@ class _EnvVariables(DataPool, ABC):
     olm_operators: EnvVar = EnvVar(
         ["OLM_OPERATORS"], loader=lambda operators: re.split(r"\s|,", operators.lower()), default=[]
     )
+    custom_manifests: EnvVar = EnvVar(
+        ["CUSTOM_MANIFESTS_FILES"],
+        loader=lambda files: [get_manifest_from_path(Path(f)) for f in re.split(r"\s|,", files)],
```

Comment:
what about discovering the files if "CUSTOM_MANIFESTS_FILES" points to a directory? I think it would be easier from user point of view, instead of listing each file.

Reply:
I wanted to mimic the service and UI behavior, but we can add a folder

Reply:
I think it would be great to be able to add a folder, it's also what `aicli` does